src/Render/Render.jl

```diff
@@ -7,4 +7,33 @@ Render
 
 using Compat
 
+import ..Elements: Node
+
+
+immutable EmptyRenderError <: Exception
+    msg :: AbstractString
+end
+
+function checkchildren(n::Node)
+    isempty(n.children) && throw(EmptyRenderError(string("'$(typeof(n))': '",
```

Comment:
``typeof(n)`` could be avoided by adding a typevar, ``checkchildren{T}(n::Node{T})``, to the method's signature. Then using ``T`` instead.